keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -208,12 +208,13 @@ Additional tests will be added to this file to cover the garbage collection e2e.
 
 - Configuration field added to the Kubelet (disabled by default)
 - Feature supported by Kubelet Image Manager
-- Unit tests and e2e tests added
+- Unit tests
 - Add a metric `kubelet_image_garbage_collected_total` which tracks the number of images the kubelet is GC'ing through any mechanism.
 
 #### Beta
 
 - Gather feedback from users
```

Comment:
do you think this will be met in 1.30 as a beta requirement? Will we be able to collect feedback from users?

Reply:
yeah tbh we don't really have a lot of feedback. I removed it as a requirement, does that make sense to do?